aws/cloudformation.yml

```diff
@@ -231,6 +231,9 @@ Resources:
                           - ecs:UpdateService
                           - ecs:CreateService
                           - ecs:DeleteService
+                          - ecs:RegisterTaskDefinition
+                          - ecs:DeregisterTaskDefinition
+                          - ecs:DescribeTaskDefinition
```

Comment:
@deorus these 3 cannot be applied to the service, so this change won't have any effect. For these 3 added actions the only resource supported is '*', so you need to add this as a new statement.